salesagility/SuiteCRM
# Fixing header and pagination in the Detail View

Fixing header in the detail view for SuiteP.

## Description




- As A User I want Less spacing at the top of Detail view and a consistent design
- Given a user on Desktop Detail View the next and previous button will show inline action menu, as shown in the mock up and the buttons will be styled inline with the mock up.
- Given a user on Desktop Detail View the Spacing between the record title and the Action menu will be reduced to a sensible level, in line with the rest of the design, contrary to the mock up.
- Given a user on Desktop Detail View I want the Favourites Star to show inline with the mock up.

## Motivation and Context

- Changes are more compliant with the SuiteP design

## How To Test This

- Goto the list view in accounts
- Select a few records
- Click on a link for a single record

You should see the following

Mobile

![screenshot_20161207_135028](https://cloud.githubusercontent.com/assets/12231216/20970238/810a9fe2-bc84-11e6-88c6-09e16c902fca.png)

Tablet
![screenshot_20161207_135013](https://cloud.githubusercontent.com/assets/12231216/20970246/8604beba-bc84-11e6-9f01-8ddf2f22b264.png)

Desktop
![screenshot_20161207_135003](https://cloud.githubusercontent.com/assets/12231216/20970243/845b0344-bc84-11e6-9bfa-1153f44fe22c.png)

## Types of changes


### Final checklist
